Web/starter_project_vanguard/pages/blog/[id].tsx

```diff
@@ -0,0 +1,81 @@
+import Image from 'next/image'
+import { Blog } from '../../types/blog/blog'
+import blogs from '../../data/blogs.json'
+import { useRouter } from 'next/router'
+import Related from '@/components/blog/RelatedBlog'
+
+const SingleBlog = () => {
+  const router = useRouter()
+  let { id } = router.query
+
+  const blog = blogs.filter((item: Blog) => item._id === id)[0]
+  
+  return (
+    <div className="w-3/4 m-auto">
+      <div className="text-center mt-10 flex-col">
+        <h1 className="text-4xl font-bold font-poppins">
+          {blog.title}
+        </h1>
+
+        <p className="mt-4 tracking-widest font-montserrat">
+          PROGRAMMING, TECH | {blog.time} MIN READ
+        </p>
+
+        <Image
+          className="mt-10 w-full h-96 object-cover ml-auto mr-auto"
+          src={blog.img}
+          width={400}
+          height={100}
+          alt="hello"
+        />
+      </div>
+
+      <div className="card text-center mt-10">
+        <Image
+          className="w-16 h-16 bg-orange-100 m-auto object-cover rounded-full "
+          src={blog.authorPhoto}
+          width={200}
+          height={200}
+          alt="card"
+        />
+        <div className="font-montserrat mt-5">
+          <h4 className="font-montserrat uppercase">
+            {blog.authorName} | {blog.profession}
+          </h4>
+          <h6 className="text-blue-500 font-montserrat uppercase">
+            @{blog.authorUserName}
+          </h6>
+        </div>
+      </div>
+
+      <div className="blog text-center m-4 md:m-16 mt-10  ">
+        <h2 className="m-auto text-2xl mb-8 text-left font-poppins">
+          {blog.descriptionTitle}
+        </h2>
+
+        <div className="font-montserrat text-sm text-gray-400 flex  flex-col gap-5 ">
+          <p className="text-left">
+            {blog.description} <br />
+            <br /> {blog.description} <br />
+            <br /> {blog.description}
```

Comment:
This isn't the best way to do this, you should expect blog.description to be one large block of text, and then you can separate it into multiple paragraphs by splitting using the newline character (\n).